static/js/sensor_connection.js

```diff
@@ -31,7 +37,7 @@ function sendToSensor() {
 var checkConInterval = 100; //time interval between checks for websocket connection
 var socket;
 function init() {
-  socket = new WebSocket("ws://" + "192.168.205.85" + ":81/");
+  socket = new WebSocket("ws://" + "192.168.2.127" + ":81/");
```

Comment:
I guess things like this should not be hardcoded. But I guess it is fine.

Reply:
Still not sure how I will implement IP's in the final product, will the user have the access to enter the ip manually or should the admin assign them before